_get_started/installation/windows.md

```diff
@@ -18,7 +18,7 @@ PyTorch is supported on the following Windows distributions:
 ### Python
 {: #windows-python}
 
-Currently, PyTorch on Windows only supports Python 3.x; Python 2.x is not supported.
+Currently, PyTorch on Windows only supports Python 3.7-3.9; Python 2.x is not supported.
```

Comment:
I think 1.11 will be released with python 3.10 (cc @seemethere to confirm). Maybe we can put that here directly to avoid having to change it again?

Reply:
Correct. I am working on 3.10 update currently, however I would prefer to do one more update. When 3.10 is ready to be released

Reply:
Ok!